content/blog/2025/02/2025-02-14-building-a-safer-matrix.md

```diff
@@ -136,23 +136,23 @@ Die überwiegende Mehrheit der Matrix-Nutzung ist konstruktiv: Sie ermöglicht Z
+Der größte Teil unserer Mittel als Stiftung wird für unser Vollzeit-Trust-&-Safety-Team verwendet, und wir haben dieses Engagement Ende 2024 erweitert. Das Team kümmert sich täglich um die Sichtung, Untersuchung, Identifizierung und Entfernung schädlicher Inhalte vom Matrix.org-Server und entfernt Nutzer, die dieses Material teilen. Darüber hinaus entwickelt es Werkzeuge zur Verhinderung, Erkennung und Entfernung schädlicher Inhalte und zum Schutz der Mitarbeiter, die an den Berichten und Untersuchungen der Nutzer arbeiten.
 
 Die Menschen, die das Trust & Safety Team der Stiftung bilden, sind engagierte Fachleute, die jeden Tag ihre eigene geistige Gesundheit und ihr Wohlbefinden aufs Spiel setzen, indem sie schädliche Inhalte überprüfen, die von Menschen hinzugefügt wurden, die den von uns angebotenen Dienst missbrauchen. Bei ihrer Arbeit sind sie Gefahren ausgesetzt, darunter sexuelle Ausbeutung und Missbrauch von Kindern (CSEA), terroristische Inhalte, nicht einvernehmliche intime Bilder (NCII), Belästigung, Hass, Deepfakes, Betrug, Fehlinformationen, illegale Pornografie, Drogen, Schusswaffen, Spam, Selbstmord, Menschenhandel und mehr. Es ist eine Wäscheliste mit dem Schlimmsten, was die Menschheit zu bieten hat. Die düstere Realität ist, dass sich alle Online-Dienste mit diesen Problemen auseinandersetzen und die Arbeit zur Erkennung und Entfernung dieser Inhalte mit den Rechten ihrer Nutzer in Einklang bringen müssen. Wir haben uns dieser Arbeit verschrieben und unterstützen das Trust & Safety Team nach besten Kräften \- wir sind sehr dankbar für ihre Aufopferung.
 
-### Sicherheitswerkzeuge für Matrix
+### Schutzwerkzeuge für Matrix
 
-Das Trust & Safety Team befasst sich mit dem Thema Sicherheit aus zwei Blickwinkeln: Schutz für die Nutzer des Matrix.org-Servers und Schutz für das gesamte Matrix-Netz als Ganzes. Für letzteres tragen wir zur Entwicklung der Matrix-Spezifikation bei, stehen im engen Austausch mit anderen  im Matrix-Ökosystem und beraten uns mit Regierungen, Strafverfolgungsbehörden, zivilgesellschaftlichen Gruppen, Hochschulen und Industriegruppen. Wir investieren auch in Open-Source-Tools, um das Ökosystem bei diesem Problem zu unterstützen. Für ersteres setzen wir eine Mischung aus proaktivem und reaktivem Vorgehen gegen Online-Missbrauch ein.
+Das Trust & Safety Team befasst sich mit dem Thema Schutz aus zwei Blickwinkeln: Schutz für die Nutzer des Matrix.org-Servers und Schutz für das gesamte Matrix-Netzwerk als Ganzes. Für letzteres tragen wir zur Entwicklung der Matrix-Spezifikation bei, stehen im engen Austausch mit anderen im Matrix-Ökosystem und beraten uns mit Regierungen, Strafverfolgungsbehörden, zivilgesellschaftlichen Gruppen, Hochschulen und Industriegruppen. Wir investieren auch in Open-Source-Werkzeuge, um das Ökosystem bei diesem Problem zu unterstützen. Für ersteres setzen wir eine Mischung aus proaktivem und reaktivem Vorgehen gegen Online-Missbrauch ein.
```

Comment:
consistently translated "safety" as "Schutz" and "tool" as "Werkzeug" across the whole post